cves/kernel/CVE-2013-1796.yml

```diff
@@ -266,21 +124,42 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
-ipc:
+upvotes: "4"
```

Comment:
You're not supposed to include the upvotes.

Reply:
Whoops. That was some misreading on my part.

Reply:
No worries at all, I made the same mistake lol. I'd give it 4 as well. Its a pretty understandable Vulnerability and a bit interesting imo.